pkg/scheduler/factory/factory.go

```diff
@@ -670,8 +670,10 @@ func (c *configFactory) updatePodInCache(oldObj, newObj interface{}) {
 			case schedulerapi.ResizeActionReschedule:
 				// Case 2. Node does not have capacity. Delete pod, let controller re-create pod.
 				c.recorder.Eventf(newPod, v1.EventTypeNormal, "DeletePodForResizeReschedule", "Deleting pod %s to reschedule for resizing", podName)
-				delete(newPod.ObjectMeta.Annotations, schedulerapi.AnnotationResizeResourcesActionVer)
-				if err := c.client.CoreV1().Pods(newPod.Namespace).Delete(newPod.Name, nil); err != nil {
+				delete(newPod.Annotations, schedulerapi.AnnotationResizeResourcesActionVer)
+				deleteOptions := metav1.NewDeleteOptions(0)
```

Comment:
I recall this delete option was removed in an earlier PR?

Reply:
that was a bug. it was processing delete twice (now checked correctly by DeletionTimeStamp un updatePod function.